examples/apps/echo/README.md

```diff
@@ -0,0 +1,53 @@
+# Echo Demo Application
+
+This simple demo application echos back the same message sent from
+the client to the worker.
+
+## Using the Echo Command Line Client
+
+The command line client, `echo_client.py` sends a message to the worker,
+which echos back the same message back to the client.
+For command line options, type `./echo_client.py -h` from the
+`client` directory.
+
+To use:
+
+1.  If needed, update the Ethereum account and direct registry contract
+    information in `docker/Dockerfile.tcf-dev` and
+    `examples/common/python/connectors/tcf_connector.toml`
+2.  Follow instructions in the "Docker-based Build and Execution" section of
+    the [build document](../../../BUILD.md#dockerbuild) through step 4
+    (activating a virtual environment)
+3.  Terminal 1 is running the TCF Enclave Manager and Listener with
+    `docker-compose` . Terminal 2 is running the Docker container shell
+    with the `(_dev)` prompt
+4.  In Terminal 2 run `cd $TCF_HOME/examples/apps/echo/client`
+5.  In Terminal 2 install Python3's TKInter GUI library with
+    ```bash
+    export DEBIAN_FRONTEND=noninteractive; apt update; apt -y install python3-tk
```

Comment:
Is echo client using GUI library ? AFAIK only Heart Disease eval client is using GUI.
python3-tk  is already installed in docker. Is this instruction for standalone build ?

Reply:
@manojgop You are correct. The echo client is not using the TK GUI library, so I removed #5 above. Thanks.